tests/test_io.py

```diff
@@ -47,77 +76,44 @@ def test_load_scan_spm(load_scan: LoadScan) -> None:
 #         ("nm", 50, 50, 0.048828125),
 #     ],
 # )
-# def test_extract_pixel_to_nm_scaling(load_scan: LoadScan, unit, x, y, expected) -> None:
+# def test_extract_pixel_to_nm_scaling(load_scan: LoadScans, unit, x, y, expected) -> None:
 #     """Test extraction of pixels to nanometer scaling."""
 #     load_scan.load_spm()
 #     load_scan._spm_pixel_to_nm_scaling() {"unit": unit, "x": x, "y": y}
 #     test_filters_random.extract_pixel_to_nm_scaling()
 #     assert test_filters_random.pixel_to_nm_scaling == expected
 
 
+def test_load_scan_extract_jpk() -> None:
+    """Test extraction of data from loaded JPK image."""
+    assert True
```

Comment:
Should this test be completed before merging?

Reply:
Good catch! It can be removed as its parameterised in the tests below. I think it slipped through the net when merging branches due to it previously being a place holder.